src/NzbDrone.Core/Indexers/Newznab/NewznabRequestGenerator.cs

```diff
@@ -116,7 +117,7 @@ public virtual IndexerPageableRequestChain GetSearchRequests(SceneSearchCriteria
                     pageableRequests.Add(GetPagedRequests(MaxPages,
                         Settings.Categories,
                         "search",
-                        $"&q={NewsnabifyTitle(queryTitle)}+{releaseDate}"));
+                        $"&q={NewsnabifyTitle(queryTitle)}+{releaseDate}+{performer}"));
```

Comment:
Does this makes results too restrictive? What if the performer is not in the release name

Reply:
The reason I added it was I ran a search without the performer using 24.01.20 and it returned 2 pages with dates ranging from 24.01.20-20.01.24
I can add a check that if no results are returned and run the search again without the performer for each query. Or add additional criteria to the queryTitles like title+performer, site+title, site+performer. I don't know if the indexer will see it as a ddos attack.

Reply:
Gonna need to do some more testing. I searched for a known release without the performer in the title and we still rejected the release with the original query. Did you want to exclude this commit from the merge for now?